setup.py

```diff
@@ -47,6 +47,7 @@
     entry_points={'console_scripts': [
         'django-admin = django.core.management:execute_from_command_line',
     ]},
+    install_requires=['pytz > dev'],
```

Comment:
This is to fix some problem with pip 1.4 (a9589dd280ba84db3131f32b423ef45c67a5a236). Maybe we don't have to worry about that anymore.